ScalaDays/Extension/Array+Extension.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+extension Array {
+    
+    public subscript(safe index: Int) -> Element? {
+        guard index < count else { return nil }
+        return self[index]
+    }
```

Comment:
This is how the standard library should have always been... I'd only add:
```suggestion
    public subscript(safe index: Int) -> Element? {
        guard index >= 0 && index < count else { return nil }
        return self[index]
    }
```

Reply:
Good catch!